recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ source:
     - 5eff1ecc8413b0dc93a1ab047d7fed751e6cb40e.patch
 
 build:
-  number: 3
+  number: 4
```

Comment:
```suggestion
  number: 3
```

Reply:
Are you sure that the hash did not change for the other python versions?

Reply:
I checked 2-3 builds and they had the same hash. Not sure what could conceivably change the hashes here.

Reply:
Checked a couple more now across all platforms, and couldn't find a divergence.